src/website/views/templates/order-checkout-flow.pug

```diff
@@ -0,0 +1,1697 @@
+                        <label for="4" class="">City</label>
+                      </div>
+                      <div class="a-inputWrapper -floatingLabel -mb--1 -mr--1 -w--50" id="floating-label-5">
+                        <select class="a-input -xl -mr--1" id="floating-label-select">
+                          <option></option>
+                          <option>State 1</option>
+                          <option>State 2</option>
+                        </select>
+                        <label for="floating-label-select" class="">Select State</label>
+                      </div>
+                      <div class="a-inputWrapper -floatingLabel -mb--1" id="floating-label-6">
+                        <input class="a-input -xl -mr--3" type="text" id="i5" style="max-width: 10rem;">
+                        <label for="6">Zip</label>
+                      </div>
+                    </div>
+                  </div>
+                  <div class="m-epanel__footer -justify-content--end">
+                    <button class="a-btn -lg -brand -pill -ml--2" data-chi-epanel-action="next" id="location">
+                      Continue
+                    </button>
+                  </div>
+                </div>
+              </div>
+            </div>
+          </div>
+          <!--Packages Step-->
+          <div class="m-epanel -disabled" data-chi-epanel-group="checkout">
+            <div class="m-epanel__header">
+              <span class="m-epanel__number">2</span>
+              <h4 class="m-epanel__title">Packages</h4>
+              <div class="m-epanel__content">
+                <div class="m-epanel__collapse">
+                  <div class="-done--only -text">Team (x50)</div>
+                </div>
+              </div>
+              <div class="m-epanel__action -done--only">
+                <button class="a-btn -brand -flat -pill" data-chi-epanel-action="active">Change</button>
+              </div>
+            </div>
+            <div class="m-epanel__collapse">
+              <div class="-active--only">
+                <div class="m-epanel__body">
+                  <div class="m-epanel__content">
+                    <div class="-d--flex -flex--row -justify-content--between">
+                      <h5 class="-mb--3">Add packages</h5>
+                      <a class="-text--small -text--brand" href="#">Compare features ›</a>
+                    </div>
+                    <div class="packages -pb--2">
+                      <div class="a-grid -mb--3">
+                        <div class="a-col">
+                          <div class="a-card" id="1" style="cursor: pointer;">
+                            <div class="a-card__content -text--center -py--3">
+                              <i class="a-icon icon-circle-info-outline -sm -text--muted -position--absolute" style="top:15px;right:15px;"></i>
+                              <h5 class="-mb--3">Calling</h5>
```

Comment:
The font-size and spacing here doesn't quite match the design. Let's change each package title from h5 to h6 and replace the current margin class with `-mt--0`

Reply:
done

Reply:
done